docs/topics/logging.rst

```diff
@@ -193,6 +193,45 @@ to override some of the Scrapy settings regarding logging.
     Module `logging.handlers <https://docs.python.org/2/library/logging.handlers.html>`_
         Further documentation on available handlers
 
+Custom Log Formats
+-------------------
+
+Custom log format can be set for different actions by extending ``scrapy.logformatter.LogFormatter`` class.
```

Comment:
I think 

    :class:`~scrapy.logformatter.LogFormatter`

yould be better than

    ``scrapy.logformatter.LogFormatter``

specially now that you have provided API reference documentation for that class (thanks!).